test/visibility/import/edgeCases/anotherMultipleDef.chpl

```diff
@@ -0,0 +1,21 @@
+module A {
```

Comment:
This test more addresses my concern number 1 in that comment.  Concern number 2 is more about if you have two re-exporting imports that bring in symbols with the same name to the same scope, and then try to use that name.  E.g.

```chapel
module A {
  var x = 3;
}
module B {
  var x = 7;
}
module C {
  public import A.x;
  public import B.x;
}
module D {
  use C; // or import C for C.x

  proc main() {
    writeln(x); // Should be a conflict, which one should it pick?
  }
}
```